dgl_sparse/src/python_binding.cc

```diff
@@ -33,7 +33,9 @@ TORCH_LIBRARY(dgl_sparse, m) {
       .def("transpose", &SparseMatrix::Transpose)
       .def("coalesce", &SparseMatrix::Coalesce)
       .def("has_duplicate", &SparseMatrix::HasDuplicate)
-      .def("is_diag", &SparseMatrix::HasDiag);
+      .def("is_diag", &SparseMatrix::HasDiag)
+      .def("rowwise_select", &SparseMatrix::RowwiseSelect);
```

Comment:
Install `clang-format` in Python environment and use vscode to format your C++ code.

Reply:
Applied.